### PR TITLE
Tls diagrams

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -1206,6 +1206,64 @@ of handshake data starting from an offset of 0.
...
 
+{{tls-1rtt-handshake}} provides an overview of the 1-RTT handshake.
+Each line shows a QUIC packet with the packet type and packet
+number shown first, followed by the contents. So, for instance
+the first packet is of type Initial, with packet number 0, and
+contains a CRYPTO_HS frame carrying the ClientHello.
+
+Note that multiple QUIC packets may be coalesced into a single
+UDP datagram (see {{packet-coalesce}}, and so this handshake
+may consist of anywhere from 4 to 9 UDP datagrams. Moreover,
```
Comment (ianswett) [inline draft-ietf-quic-transport.md]: This "Moreover, " sentence seems redundant to the above sentence, so I'd remove it.

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -1206,6 +1206,64 @@ of handshake data starting from an offset of 0.
 
 Details of how TLS is integrated with QUIC are provided in {{QUIC-TLS}}.
```
Comment (ianswett) [inline draft-ietf-quic-transport.md]: I'd like a header for this section.

Comment (ekr) [reply draft-ietf-quic-transport.md]: Any thoughts?

Comment (ianswett) [reply draft-ietf-quic-transport.md]: Example Handshake Flow?

**Pull request merged**